src/components/date-range/calendars.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./calendars.scss";
+import { Calendar } from "./calendar";
+import { useStateContext } from "../../hooks/use-state";
+
+interface ICalendarsProps {
+  selectedCalendar: string|undefined;
```

Comment:
Just for consistency
```suggestion
  selectedCalendar: string | undefined;
```